Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3
+
+RUN apt-get update && apt-get upgrade -y && \
+    pip install docquery && \
+    apt-get install -qq -y tesseract-ocr poppler-utils
```

Comment:
can we add: "add-apt-repository ppa:alex-p/tesseract-ocr5" to get tesseract ocr5

Reply:
@amazingvince is this instead of `apt-get install tesseract-ocr` ?